src/hdusd/usd_nodes/nodes/usd_file.py

```diff
@@ -66,6 +67,35 @@ def compute(self, **kwargs):
 
         input_stage = Usd.Stage.Open(file_path)
 
+        def _resolve_world_path(light_obj):
+            tex_attr = light_obj.GetTextureFileAttr()
+            src_filepath = tex_attr.Get()
+            if src_filepath:
+                tex_attr.Set(src_filepath.resolvedPath)
+
+        for prim in input_stage.TraverseAll():
```

Comment:
Mind if it can be moved to separated function with [link](https://github.com/GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon/pull/239/files#diff-902ab034ad9bab97e9478ed50d4f5b014baf47bf96889e004392a2fc9ee5760eR465)